pkg/flare/archive_win.go

```diff
@@ -209,3 +210,52 @@ func (p permissionsInfos) add(filePath string) {}
 func (p permissionsInfos) commit(tempDir, hostname string, mode os.FileMode) error {
 	return nil
 }
+
+func zipServiceStatus(tempDir, hostname string) error {
+	f := filepath.Join(tempDir, hostname, "servicestatus.txt")
+	err := ensureParentDirsExist(f)
+	if err != nil {
+		log.Warnf("Error in ensureParentDirsExist %v", err)
+		return err
+	}
+
+	fh, err := os.Create(f)
+	if err != nil {
+		log.Warnf("Error creating temp file %s %v", f, err)
+	}
+	defer fh.Close()
+	cancelctx, cancelfunc := context.WithTimeout(context.Background(), execTimeout)
+	defer cancelfunc()
+
+	cmd := exec.CommandContext(cancelctx, "powershell", "-c", "get-service", "data*,ddnpm", "|", "fl")
+
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err = cmd.Run()
+	if err != nil {
+		log.Warnf("Error running powershell command %v", err)
+		// for some reason the lodctr command returns error 259 even when
```

Comment:
Can we check for this specific error and ignore it? Returning otherwise?

Reply:
What is `lodctr`?

Reply:
lodctr is the windows tool that loads/displays the counter strings.  And, since I've found it to be somewhat inconsistent, I thought it better to just keep going regardless of the output.  We're more likely to get the info we want, and if it logs some error string we're not worse off than skipping it.